src/schematools/importer/ndjson.py

```diff
@@ -92,6 +92,10 @@ def parse_object(self, source: dict) -> dict[str, list[Record]]:
         for field in self.dataset_table.get_fields(include_subfields=True):
             if field.id == "schema" and field.type.startswith("https://"):
                 continue
+            if field.id == "id" and self.dataset_table.has_composite_key:
```

Comment:
Needs to use the same condition that `_fill_composite_pk` uses:

```suggestion
            if field.id == "id" and self.dataset_table.is_autoincrement and self.dataset_table.has_composite_key:
```